.env.example

```diff
@@ -1,2 +1,3 @@
 SUPABASE_URL=
-SUPABASE_ANON_KEY=
+SUPABASE_REF=
```

Comment:
What is this? There should always be documentation around environment variables, ideally in the README; as this affects setup and the ability for others to work on this. 

Although I don't see why a new env var is necessary in the first place.

Reply:
I've removed it and just parsing it from the url instead